pyEcoHAB/following.py

```diff
@@ -10,11 +10,13 @@
 from .write_to_file import write_binned_data
 from .write_to_file import write_interpair_intervals
 from .write_to_file import write_bootstrap_results
+from .write_to_file import write_sum_data
 from .plotting_functions import single_in_cohort_soc_plot, make_RasterPlot
 from .plotting_functions import pooled_hists
 from .plotting_functions import make_histograms_for_every_mouse
 from .plotting_functions import pooled_hists_for_every_mouse
 from .plotting_functions import single_histogram_figures
+from collections import OrderedDict
```

Comment:
According to good practices imports from modules included in Python distribution should be on top. Please move.